### DISCUSSION TITLE
Scalafix for 0.22 and 1.0: what do we need?

### DISCUSSION THREAD

**Discussion by rossabaker** [http4s/http4s#4633] [Ideas]
We're starting to develop scalafix rules for the changes coming in 0.22 and 1.0.  It would be helpful to hear from users about what's painful in their upgrades.  If you have an 0.21 service and can get it on 0.22 or 1.0, and see recurring patterns in the breakage, please comment here, and we'll see how much we can automate for the community.

- In progress: backtick removal
- Known needed: header typeclass

What else hurts?

**Comment by ScalaWilliam** [comment]
Just to clarify, should users try to upgrade to v0.22.0-M5 or to v1.0.0-M19, and what would be the difference between the two?

**Reply by rossabaker** [reply]
* http4s-0.22 -> cats-effect-2.x.  A path to Dotty without biting off the CE upgrade.
* http4s-1.0 -> cats-effect-3.x. Where you'll want to be if your stack supports it.
